docs/CactbotCustomization.md

```diff
@@ -41,6 +43,30 @@ These options are stored in your
 file.
 You should not need to edit that file directly.
 
+## Changing Trigger Text with the cactbot UI
+
+In the cactbot configuration UI, under
+ACT -> Plugins -> OverlayPlugin.dll -> Cactbot -> Raidboss,
+there are individual trigger listings.
+You can use these listings to change various exposed configuration settings per trigger.
+
+Settings with a bell (🔔) next to their name are trigger mostly toutputs that you can override.
```

Comment:
Is this a typo? Might be best to rephrase it if not.

Reply:
Fixed, thanks.